specs/rails/db_migrate.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Apply Rails database migrations
+command: |-
+    rails db:migrate
+tags:
+  - rails
+description: Uses the rails rake task for applying all the migrations
```

Comment:
Can we use active voice here:
"Uses the rails rake task to apply all the database migrations"